### PR TITLE
Listener for do not enable setting

### PR DESCRIPTION
**Pull request by luyaojchen**

### CODE REVIEW

**Comment by luyaojchen** [comment]
image

no notifications if setting is enabled, i stop notifying once thats sent. It checks every 10 seconds if the setting is enabled or disabled.

File: browser-extension/public/background.js [hunk 1]
```diff
@@ -34,3 +34,56 @@ chrome.tabs.onUpdated.addListener((tabId, changeInfo, tab) => {
...
+        notify = true;
+        chrome.notifications.create("", {
+          type: "basic",
+          iconUrl: "bytebucks48.ico",
+          title: "Do Not Track",
+          message:
+            "Do Not Track is disabled. Please enable it in your Chrome settings for enhanced privacy.",
+          priority: 2,
+        });
+        setTimeout(() => {}, 30000); // im giving them 30 seconds to turn it on
```
Comment (deandrebaker) [inline browser-extension/public/background.js:76]: What does this line do? As far as I know, `setTimeout` will run whatever callback you give it in the background after the time you specified, but the callback is empty.

File: browser-extension/public/manifest.json [hunk 2]
```diff
@@ -22,6 +22,8 @@
     "activeTab",
     "scripting",
     "webRequest",
-    "webNavigation"
+    "webNavigation",
+    "privacy",
+    "notifications"
     ]
```
Comment (deandrebaker) [inline browser-extension/public/manifest.json:28]:
```suggestion
  ]
```

**Review by deandrebaker — COMMENTED** [review]
Can you take a screenshot of what is supposed to show up?

File: browser-extension/public/background.js [hunk 3]
```diff
@@ -73,7 +73,9 @@ function checkDNTSetting() {
             "Do Not Track is disabled. Please enable it in your Chrome settings for enhanced privacy.",
           priority: 2,
         });
-        setTimeout(() => {}, 30000); // im giving them 30 seconds to turn it on
+        setTimeout(() => {
+          notify = false;
```
Comment (luyaojchen) [inline browser-extension/public/background.js:77]: @deandrebaker I filled it in, so the flow is if the setting is disabled, we will remind them every minute if they still haven't turned it on. 

I set notify to true in the beginning so it doesn't spam the notifs, but after the delay the flag will get reset

Comment (deandrebaker) [reply browser-extension/public/background.js:77]: Okay, sounds good

**Review by deandrebaker — APPROVED** [review]

**Pull request merged**